syntax = "proto2";

package mmkv;

message NodeConf
{
  optional uint32 port       = 1; // sharder port
  required string host       = 2;
  optional uint32 mmkvd_port = 3;
  repeated uint64 shard_ids  = 4;
}

message Configuration { map<uint64, NodeConf> node_conf_map = 1; }
